proxy/http/HttpSM.cc

```diff
@@ -2451,6 +2451,12 @@ HttpSM::state_cache_open_write(int event, void *data)
   // INTENTIONAL FALL THROUGH
   // Allow for stale object to be served
   case CACHE_EVENT_OPEN_READ:
+    if (!t_state.cache_info.object_read) {
+      ink_assert(t_state.cache_open_write_fail_action == HttpTransact::CACHE_WL_FAIL_ACTION_READ_RETRY);
+      t_state.cache_lookup_result         = HttpTransact::CACHE_LOOKUP_NONE;
+      t_state.cache_info.write_lock_state = HttpTransact::CACHE_WL_READ_RETRY;
+      break;
```

Comment:
Why can `t_state.cache_info.object_read` be nullptr ?

Reply:
I see `state_cache_open_write ` call OPEN_READ to sm.

Reply:
Yes, that's correct. That's to identify a doc_busy indirectly caused by a concurrent request to the same cache (using write lock failure signal).